rest_authenticate/tests/test_rest_session.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Wakari SRL
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import odoo.tests.common
+from odoo.http import root
+from odoo.tests import new_test_user
+from odoo.tests.common import HttpCase
+
+HOST = "127.0.0.1"
+PORT = odoo.tools.config["http_port"]
+
+
+@odoo.tests.common.at_install(False)
+@odoo.tests.common.post_install(True)
+class TestSessionRestAPI(HttpCase):
+    def setUp(self):
+        super().setUp()
+        self.password = "test_password"
+        self.user = new_test_user(self.env, login="test_user", password=self.password)
+
+    @odoo.tools.mute_logger("odoo.addons.base_rest.http")
+    def test_01(self):
+        """Test authentication"""
+        params = {
+            "db": self.env.cr.dbname,
+            "login": self.user.login,
+            "password": self.password,
+        }
+        url = "http://{}:{}{}".format(HOST, PORT, "/session/auth/login")
```

Comment:
you don't need to compute the URL yourself, just pass the path

Reply:
That's only true if I call `url_open`, but in that case I cannot control the http method to use -- no `POST` with an empty body for example.

Reply:
oh, you are right. Never mind then :)